api/provider/logger/main.go

```diff
@@ -10,5 +10,5 @@ type LogFactory interface {
 	Warn(...string)
 	Print(...string)
 	Err(...string)
-    Note(...string)
+	Note(...string)
```

Comment:
What is this for?

Reply:
For note messages like the prompt where it displays important information